src/main/java/com/karl/http/HTTPHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.karl.http;
+
+import java.io.BufferedReader;
+import com.karl.constants.Globals;
+import com.karl.wrappers.WrappedSocket;
+
+public class HTTPHandler implements Runnable {
+  public WrappedSocket socket;
+
+  public HTTPHandler(WrappedSocket socket) {
+    this.socket = socket;
+  }
+
+  public void run() {
+    try {
+      final BufferedReader requestReader = socket.getReader();
+      String requestString = "";
+      while (requestReader.ready()) {
+        requestString = requestString + requestReader.readLine();
```

Comment:
Replacing the `+` concatenation with `StringBuilder.append(requestReader.readLine())` is going to be a lot more performant.

Reply:
No doubt 👍